src/components/AppStructure.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+import Header from './Header.jsx';
+import Footer from './Footer.jsx';
+
+export default class AppStructure extends PureComponent {
+  render() {
+    return (
+      <div>
+        <Header />
```

Comment:
You have to add `{ this.props.children }` in between `<Header />` and `<Footer />`. This is because `AppStructure` is used at the top level `Route`. So when you go to the URL `/surveys` (for example), it will render the component `AppStructure`, and pass to it as props, the children components of that `Route`, which in this example would be `AllSurveysContainer`. So we need to make sure that `AllSurveysContainer` is rendered by `AppStructure`. Note that the `children` prop automatically gets passed to `AppStructure` by `react-router`.

Also, when adding `{ this.props.children }`, wrap it in a `div` and give it some styling so that the body in between the header and footer has some margin.

Also, you will need to add `children` to the `propTypes`:
```(js)
  static propTypes = {
    children: PropTypes.object,
  }
```